### DISCUSSION TITLE
29th open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#4556] [General]
Hi everyone,

The rust-libp2p maintainers will meet tomorrow (Tuesday) 09:00 UTC. See [libp2p public events calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item or list it in [our notes document](https://hackmd.io/xoHKlFuER0OVj1DGDuFROw).

See you all tomorrow!

**Comment by mxinden** [comment]
Maybe @altonen you are interested in presenting https://github.com/altonen/litep2p/?

**Reply by altonen** [reply]
Yeah sure, thanks for the opportunity
